tests/integration/golang/mlflow/namespace/namespace_test.go

```diff
@@ -32,96 +28,6 @@ func (s *NamespaceTestSuite) TearDownTest() {
 	require.Nil(s.T(), s.NamespaceFixtures.UnloadFixtures())
 }
 
-func (s *NamespaceTestSuite) Test_Ok() {
-	tests := []struct {
-		name      string
-		setup     func() *models.Experiment
-		namespace string
-	}{
-		{
-			name: "TestCustomNamespaces",
-			setup: func() *models.Experiment {
-				namespace, err := s.NamespaceFixtures.CreateNamespace(context.Background(), &models.Namespace{
-					Code:                "newly-created-namespace",
-					DefaultExperimentID: common.GetPointer(int32(0)),
-				})
-				require.Nil(s.T(), err)
-				experiment, err := s.ExperimentFixtures.CreateExperiment(context.Background(), &models.Experiment{
```

Comment:
stupid and unnecessary test - just removed it. we test absolutely the same in `experiment` flow tests.